### PR TITLE
Update README.md with new sentence of the poem

### PR DESCRIPTION
**Pull request by denigerar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Poem
 This is an open source poem. All contributions would be great!
+Github is a program I will treat
```
Comment (JanSawicki) [inline README.md:3]: Very nice reference to the topic of the training and the rhyme 😄

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 Nice poem mate, I rate 8/8!
+
```
Comment (JanSawicki) [inline README.md:5]: This line is not needed.

**Review by JanSawicki — APPROVED** [review]
I really like this pull request. It looks really **cool**.

File: README.md (hunk 1 above)
Comment (denigerar) [reply README.md:3]: Thank you!

File: README.md (hunk 2 above)
Comment (denigerar) [reply README.md:5]: Ho, do I have deleted this line?

**Pull request closed without merging**